spec/requests/api/link_spec.rb

```diff
@@ -107,12 +107,12 @@
           }
         }
 
-        interaction_1 = FactoryGirl.create(:interaction, label: 'report', lgil_code: 1)
-        interaction_2 = FactoryGirl.create(:interaction, label: 'appeal', lgil_code: 2)
-        service_interaction_1 = FactoryGirl.create(:service_interaction, service: service, interaction: interaction_1)
-        service_interaction_2 = FactoryGirl.create(:service_interaction, service: service, interaction: interaction_2)
-        FactoryGirl.create(:link, local_authority: local_authority, service_interaction: service_interaction_1)
-        FactoryGirl.create(:link, local_authority: local_authority, service_interaction: service_interaction_2)
+        interaction_1 = create(:interaction, label: 'report', lgil_code: 1)
+        interaction_2 = create(:interaction, label: 'appeal', lgil_code: 2)
+        service_interaction_1 = create(:service_interaction, service: service, interaction: interaction_1)
+        service_interaction_2 = create(:service_interaction, service: service, interaction: interaction_2)
+        create(:link, local_authority: local_authority, service_interaction: service_interaction_1, url: "http://blackburn.example.com/abandoned-shopping-trolleys/report")
```

Comment:
these tests were asserting on a specific URL that was being generated in the Link Factory. When we changed the factory class, these unrelated tests started failing, so we updated them in the simplest possible way. Next time anyone is in the codebase in this vicinity I'd recommend a rewrite TBH.